packages/sign-client/src/controllers/engine.ts

```diff
@@ -1015,6 +1028,7 @@ export class Engine extends IEngine {
           metadata: controller.metadata,
         },
         ...(sessionProperties && { sessionProperties }),
+        sessionConfig,
```

Comment:
Do we need to take into account this being possibly `null`? Why are we treating it differently to `sessionProperties` here?